src/extension-consumer/src/consume/mod.rs

```diff
@@ -34,6 +48,10 @@ pub struct ConsumeLogOpt {
     #[structopt(short = "d", long)]
     pub disable_continuous: bool,
 
+    /// Print records in "[key] value" format, with "[null]" for no key
+    #[structopt(short, long)]
+    pub key_value: bool,
```

Comment:
actually, this flag is confusing.  it sounds like it is about consuming.  it should be more like: `--print-kv-header`.  Also printing key should be default so in this case, it should be `-disable-print-kv-header`

Reply:
Originally I had made the consumer print keys by default, but @ajhunyady said he preferred otherwise. I don't have a preference one way or the other. Let me know what you guys decide.